Core.DomainModel/ItSystemUsage/ItSystemUsage.cs

```diff
@@ -858,12 +859,17 @@ public ItSystemUsageValidationResult CheckSystemValidity()
 
             var dateErrors = CheckDatesValidity(today).ToList();
             var validAccordingToStatus = CheckLifeCycleValidity();
+            var validAccordingToContract = CheckContractValidity();
```

Comment:
it's a maybe<error>... the variable name is named "validAccordingToContract",,,, bad naming
Same goes for the one in line 861